tests/unittest_modutils.py

```diff
@@ -179,7 +179,7 @@ def test_load_packages_without_init(self) -> None:
         https://github.com/PyCQA/astroid/issues/1327
         """
         tmp_dir = Path(tempfile.gettempdir())
-        self.addCleanup(os.chdir, os.curdir)
+        self.addCleanup(os.chdir, os.getcwd())
```

Comment:
We need to actually find the cwd before changing dir, otherwise this cleanup won't do anything.